packages/shopify-app-remix/src/auth/helpers/redirect.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  TypedResponse,
+  redirect as remixRedirect,
+} from '@remix-run/server-runtime';
+
+import {BasicParams} from '../../types';
+
+import {renderAppBridge} from './render-app-bridge';
+import {getSessionTokenHeader} from './get-session-token-header';
+
+export type RedirectTarget = '_self' | '_blank' | '_parent' | '_top';
```

Comment:
Seems like we wouldn't want folks to use `_blank`, since it'll likely be blocked by popup blockers. `_parent` and `_top` are identical codepaths and both just trigger AB Remote Redirect. That leaves `_self`, which seems like it's identical to a normal Remix `redirect()`.

Maybe it would make sense to only provide a helper that redirects Admin, and leave normal HTTP redirects to Remix?


```js
import {redirect} from '@remix-run/server-runtime';

export function loader({request}) {
  const admin = authenticate.admin(request);

  // normal redirects: (for outside of Admin or in-iframe redirects)
  throw redirect('/foo');

  // what's provided here: (for redirecting Admin)
  throw admin.redirect('/foo');
}
```